LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 ml5
+Copyright (c) 2020 ml5
```

Comment:
Based on my understanding, it looks like we don't need to update the license date as this year references the start date of the copyright. More on this here: https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year.